packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -618,6 +618,13 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
         setCenter(center);
     }, [center]);
 
+    /**
+     * Sets app title based on app config.
+     */
+    useEffect(() => {
+        document.title = appConfig?.appSettings?.title ?? 'MapsIndoors Web';
```

Comment:
I disagree that this should be done at all.

What if a customer is using the Map Template web component as part of a larger web app app? Then we come along and change the title, and there's nothing they can do about it.

I believe the manipulation of the title tag belongs in the hosted web app and only there.